core/domain/summary_services.py

```diff
@@ -313,6 +399,32 @@ def get_displayable_exp_summary_dicts(exploration_summaries):
 
 
 def _get_displayable_collection_summary_dicts(collection_summaries):
+    """Returns a list with all collection summary objects from collection
```

Comment:
"collection summaries" is not a domain object. Use a similar docstring to the function above for explorations.